builds/misc/packages-release.yaml

```diff
@@ -265,7 +265,14 @@ stages:
             PACKAGE_FILTER: $(identityServicePackageFilter)
             DOWNLOAD_PATH: $(System.ArtifactsDirectory)/$(artifactName)
             IDENTITY_SERVICE_COMMIT: $(aziotis.commit)
-        - task: SFP.build-tasks.custom-build-task-1.EsrpCodeSigning@1
+        - task: AzureCLI@2
+          displayName: Setup Package Publisher
+          inputs:
+            azureSubscription: $(az.subscription)
+            scriptType: bash
+            scriptPath: $(Build.SourcesDirectory)/scripts/linux/publishReleasePackages.sh
+            arguments: --setup-pmc-only true -w "$(System.ArtifactsDirectory)" -d "$(System.ArtifactsDirectory)/$(artifactName)"
+        - task: SFP.build-tasks.custom-build-task-1.EsrpCodeSigning@3
           displayName: ESRP Binary CodeSigning (IIS)
           inputs:
             ConnectedServiceName: "Azure IoT Edge Code Sign 2"
```

Comment:
This ConnectedServiceName should be "aziotedge-pmc-v4-prod", which means we also need to cherry pick #7089.

Reply:
Done, see #7141